### ISSUE TITLE
Annoying Warning

### ISSUE THREAD

**Issue by cvladan** [gndev/shortcodes-ultimate#60]
As I'm not using ANY shortcode prefix, and probably because of that my error.log if flooded with annoying warning:

```
PHP Warning:  strpos(): Empty needle in shortcodes-ultimate-maker/includes/class-shortcodes-ultimate-maker-shortcodes.php on line 310
```

Do note that this annoyance is in Premium Add-on, but I'm not sure where to post bugs.
... It's a simple fix

**Comment by gndev** [comment]
Thank you for report. This bug will be eliminated in closest update (approx. 1-2 weeks).

**Comment by gndev** [comment]
@cvladan could you tell me what version of PHP do you use?

**Comment by gndev** [comment]
@cvladan never mind, I've found solution

**Comment by gndev** [comment]
Fixed in 1.5.6 (already available for download).
